hackqOB

```diff
@@ -0,0 +1,4453 @@
+[33mcommit 3fdf3f73e7808a1401e5db66d08ba59c6d5b785c[m[33m ([m[1;36mHEAD -> [m[1;32mcnv-3393[m[33m)[m
```

Comment:
Please drop this fine. The same for the .swp one. I recommend you creating a global gitignore with all you project files.

https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore